pkg/servicedisc/types.go

```diff
@@ -104,11 +104,12 @@ type Stats struct {
 
 // Service represents a service entry in service-discovery.
 type Service struct {
-	Addr    SWAddr       `json:"address"`
-	Type    string       `json:"type"`
-	Stats   *Stats       `json:"stats,omitempty"` // TODO: Have this implemented.
-	Geo     *GeoLocation `json:"geo,omitempty"`
-	Version string       `json:"version,omitempty"`
+	Addr     SWAddr       `json:"address"`
+	Type     string       `json:"type"`
+	Stats    *Stats       `json:"stats,omitempty"` // TODO: Have this implemented.
```

Comment:
Not part of your PR, but the basics of `Stats` has been implemented for other services already. So I guess we dont need this comment anymore?